doc/herbstluftwm.txt

```diff
@@ -519,6 +519,10 @@ rotate::
     Rotates the layout on the focused tag counterclockwise by 90 degrees. This
     only manipulates the alignment of frames, not the content of them.
 
+mirror ['--vertical'|'--horizontal']::
```

Comment:
I know that you want to use the new ArgParse functionality,
but does it make sense to have the layout mirror direction as optional flags?

A positional argument vertical/horizontal/both may also be a solution, right?
Either way is fine. But mirror without any flags is a no-op then, right?

Reply:
But playing around with it right now, pretty nice functionality anyway!

Reply:
> A positional argument vertical/horizontal/both may also be a solution, right?

Right, I never thought about that. I think `vertical`/`horizontal`/`both` (with `horizontal` being the default) is more intuitive